moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -74,33 +77,30 @@ const ProjectActivityLog = () => {
   const userList = {};
   const phaseList = {};
 
-  const {
-    getLookups,
-    lookupLoading,
-    lookupError,
-  } = useActivityLogLookupTables();
+  const { getLookups, lookupLoading, lookupError } =
+    useActivityLogLookupTables();
 
   const { loading, error, data } = useQuery(PROJECT_ACTIVITY_LOG, {
     variables: { projectId },
-    onCompleted: data => getLookups(data, "activity_log_lookup_tables"),
+    onCompleted: (data) => getLookups(data, "activity_log_lookup_tables"),
   });
 
-  const [activityId, setActivityId] = useState(null);
+  // const [activityId, setActivityId] = useState(null);
```

Comment:
i just commented out the details view in the activity log for now. If we think its not coming back at all in the near future, I can remove this code instead of just commenting out

Reply:
Cool! I'm also curious about how much this detailed view is used (if at all). I think this will be a good way to see if it is missed. 👍